acme/jose/jwk.py

```diff
@@ -120,21 +122,73 @@ def load(cls, string):
         :rtype: :class:`JWKRSA`
 
         """
-        return cls(key=util.HashableRSAKey(
-            Crypto.PublicKey.RSA.importKey(string)))
+        try:
+            key = serialization.load_pem_public_key(
```

Comment:
I moved this code to `JWK.load` and restructured a bit; Privkeys in PEM/DER are tried first, then pubkeys PEM/DER.